release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1269,6 +1269,17 @@ Optionally, you can now override the default evaluation intervals for all polici
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-configuring-pgt-compliance-eval-timeouts_ztp-advanced-policy-config[Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs].
 
+id=["specifying-the-platform-type-for-managed-clusters"]
+==== Specifying the platform type for managed clusters
+
+The Assisted Installer currently supports the following OpenShift platforms:
+
+* `BareMetal`
+* `VSphere`
+* `None`
+
+Single node OpenShift does not support `VSphere`. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#specifying-the-platform-type-for-managed-clusters_ztp-deploying-far-edge-clusters-at-scale[Specifying the platform type for managed clusters].
```

Comment:
I'm fairly sure there's an attribute for "Single node OpenShift." I think it's {sno-caps}

Reply:
Updated as suggested.